_episodes/02-tooling.md

```diff
@@ -19,6 +19,10 @@ keypoints:
 This episode describes the tools we use to build and manage lessons.
 These simplify many tasks, but make other things more complicated.
 
+## Background Resources
+
+Find out more about [setting up a lesson](https://carpentries.github.io/lesson-example/setup.html) and [using GitHub](https://github.com/carpentries/lesson-example/blob/gh-pages/CONTRIBUTING.md#using-github).
```

Comment:
We have`[lesson-setup]` link that can be used instead of `(https://carpentries.github.io/lesson-example/setup.html)`.

We also have `[contributing]` link that could be used instead of `(https://github.com/carpentries/lesson-example/blob/gh-pages/CONTRIBUTING.md).` with proper rewording, of course, because it does not have `#using-github`

Reply:
Nice! I didn't know about the stable shortcut links. I think I've got the syntax right?

Reply:
> I think I've got the syntax right?

I think so!

The links you can use are here: 
https://raw.githubusercontent.com/carpentries/lesson-example/gh-pages/_includes/links.md
That's why we have `{% include links.md %}` at the end of each episode :)